### ISSUE TITLE
[BUG]: Shared.lua:921 - Concat Error

### ISSUE THREAD

**Issue by orionshock** [HizurosWoWAddOns/Broker_Everything#34]
### WoW client

WoW Classic WotLk

### What happened?

On Login, error, Durability Module doesn't seem to update reliably.

Shared.lua:921 |  local diffStr = table.concat({link,d or 0,dM or 0},"^");

Variable "d" and "dm" are both undefined and unused (per luacheck)
Not sure what the code is trying to do here to suggest a fix.
However _G["d"] seems to be polluted by LoonBestInSlot.

### Lua Error Message?

```lua
1x Broker_Everything/shared.lua:921: invalid value (table) at index 2 in table for 'concat'
[string "=[C]"]: in function `concat'
[string "@Broker_Everything/shared.lua"]:921: in function <Broker_Everything/shared.lua:914>

Locals:
(*temporary) = <table> {
 1 = "|cffa335ee|Hitem:44296:3820:41401:40094:::::80:::::::::|h[Helm of Purified Thoughts]|h|r"
 2 = FontString {
 }
 3 = 0
}
(*temporary) = "^"
(*temporary) = FontString {
 0 = <userdata>
}
```

### Other addons?

LoonBestInSlot

**Comment by orionshock** [comment]
https://github.com/lgallucci/LoonBestInSlot/issues/13

**Comment by hizuro** [comment]
Should already be fixed since version 4.5.1.
